oct2py/io.py

```diff
@@ -49,7 +49,11 @@ def read_file(path, session=None):
 def write_file(obj, path, oned_as='row', convert_to_float=True):
     """Save a Python object to an Octave file on the given path.
     """
+    # print(obj)

```

Comment:
Can you please remove the commented out print statements here and in the test file?

Reply:
Sure!